tests/integration/test_ha.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
```

Comment:
**nitpick/todo:** Could we move these tests in to `tests/integration/ha/` please?